internal/test/go.mod

```
@@ -13,6 +13,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/kataras/iris/v12 v12.2.6-0.20230908161203-24ba4e8933b9
 	github.com/labstack/echo/v4 v4.11.3
+	github.com/oapi-codegen/nullable v1.0.0
```

I've pushed a new release (docs changes only) so worth updating too:

```suggestion
	github.com/oapi-codegen/nullable v1.0.1
```